tests/test_app_cli.py

```diff
@@ -175,8 +175,8 @@ async def test_export(tmpvenv_active_inherit: env.VirtualEnv, tmpdir, server, cl
 
     path_main_file.write(
         """
-vm1=std::Host(name="non-existing-machine", os=std::linux)
-std::ConfigFile(host=vm1, path="/test", content="")
+import std::testing
+std::testing::NullResource(name="test", agentname="agent")
```

Comment:
[Discussion](https://inmanta.slack.com/archives/CKRF0C8R3/p1715953214356089) about why this is here
> ... agentname="agent")